bento.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  name: bento
+  version: "1.9.0"
+  epoch: 0
+  description: Bento is a high performance and resilient stream processor, able to connect various sources and sinks in a range of brokering patterns and perform hydration, enrichments, transformations and filters on payloads.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 5e88f465240e6268343854377a3c5bfd9b9823e7
+      repository: https://github.com/warpstreamlabs/bento
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/ClickHouse/clickhouse-go/v2@v2.32.0
+        github.com/golang/glog@v1.2.4
+        github.com/opencontainers/runc@v1.1.14
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X main.Version=v${{package.version}}
+        -X  main.DateBuilt=$(date -u -d "@$SOURCE_DATE_EPOCH" "+%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Should we pass those as well? https://github.com/warpstreamlabs/bento/blob/fb678d80dbed042095a0296b041d48bd795d64d3/.goreleaser.yml#L18-L21

Reply:
I agree!